stdlib/public/Concurrency/Task.cpp

```diff
@@ -649,6 +641,49 @@ getAsyncClosureEntryPointAndContextSize(void *function,
           fnPtr->ExpectedContextSize};
 }
 
+SWIFT_CC(swift)
+AsyncTaskAndContext swift::swift_task_create(
+    size_t taskCreateFlags,
+    TaskOptionRecord *options,
+    const Metadata *futureResultType,
+    void *closureEntry, HeapObject *closureContext) {
+  FutureAsyncSignature::FunctionType *taskEntry;
+  size_t initialContextSize;
+  std::tie(taskEntry, initialContextSize)
+    = getAsyncClosureEntryPointAndContextSize<
+      FutureAsyncSignature,
+      SpecialPointerAuthDiscriminators::AsyncFutureFunction
+    >(closureEntry, closureContext);
+
+  return swift_task_create_common(
+      taskCreateFlags, options, futureResultType, taskEntry, closureContext,
+      initialContextSize);
+}
+
+AsyncTaskAndContext
+swift::swift_task_create_f(
+    size_t flags,
+    TaskOptionRecord *options,
+    const Metadata *futureResultType,
+    ThinNullaryAsyncSignature::FunctionType *function,
+    size_t initialContextSize) {
+  return swift_task_create_common(
+      flags, options, futureResultType,
+      function, /*closureContext=*/nullptr, initialContextSize);
+}
+
+/// Temporary hack to convert from job flags to task-creation flags,
+/// until we can eliminate the entry points that operate in terms of job
+/// flags.
+static size_t convertJobFlagsToTaskCreateFlags(size_t rawJobFlags) {
+  JobFlags jobFlags(rawJobFlags);
+  TaskCreateFlags taskCreateFlags;
+  taskCreateFlags.setPriority(jobFlags.getPriority());
+  taskCreateFlags.setIsChildTask(jobFlags.task_isChildTask());
+  taskCreateFlags.setIsAsyncLetTask(jobFlags.task_isAsyncLetTask());
+  return taskCreateFlags.getOpaqueValue();
+}
```

Comment:
add FIXME? though perhaps we'd be able to remove this still in the same PR?

Reply:
Yes, it'll go away in a subsequent commit. I should  have marked this PR as a draft.